pkg/kubectl/cmd/run.go

```diff
@@ -562,7 +623,11 @@ func generateService(f cmdutil.Factory, cmd *cobra.Command, args []string, servi
 	return nil
 }
 
-func createGeneratedObject(f cmdutil.Factory, cmd *cobra.Command, generator kubectl.Generator, names []kubectl.GeneratorParam, params map[string]interface{}, overrides, namespace string) (runtime.Object, string, meta.RESTMapper, *meta.RESTMapping, error) {
+func createGeneratedObject(f cmdutil.Factory, cmd *cobra.Command,
```

Comment:
if you persist doing that, maybe after years we will follow your style cause all the previous style functions are gone, the only inconvenience for me is every time I have to distinguish is this a style change or parameter change, but it's ok : )